tool/sdg/src/main/java/org/openscience/cdk/layout/StructureDiagramGenerator.java

```diff
@@ -581,18 +896,18 @@ private void layoutRingSet(Vector2d firstBondVector, IRingSet rs) throws CDKExce
          * this.mappedSubstructures
          */
         if (useTemplates && mappedSubstructures.getAtomContainerCount() > 0
-                && System.getProperty("java.version").indexOf("1.3.") == -1) {
+            && System.getProperty("java.version").indexOf("1.3.") == -1) {
```

Comment:
Do you have any idea why this was here? Is it still relevant? Do you think we still have people using Java 1.3??